### PR TITLE
Jekyll migration

### PR DESCRIPTION
**Pull request by lucolivier-dumaisblais**
I will port back the pre-2015 events in the coming days.

### CODE REVIEW

**Comment by lucolivier-dumaisblais** [comment]
Also, I did move the DNS of cocoaheadsmtl.com to point to the current mirror of the website on cocoaheadsmtl.github.io

**Comment by lucolivier-dumaisblais** [comment]
Ok I don't know why but I can only have one assignee .......... @Palleas @Frankacy 
I'm used to Atlassian Stash where you can have multiple assignees or am I a n00b that does not know how to use Github

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Site settings
+title: "CocoaHeads Montréal"
+email: your-email@domain.com
```
Comment (Palleas) [inline _config.yml]: Ça prendrait un mail ici

File: _config.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+description: "<strong>CocoaHeads Montréal</strong> brings together iOS and Mac developers every month for drinks and talks about anything Apple-related.
+"#> # this means to ignore newlines until "baseurl:"
+baseurl: "" # the subpath of your site, e.g. /blog/
+url: "http://cocoaheadsmtl.github.io" # the base hostname & protocol for your site
```
Comment (Palleas) [inline _config.yml]: Is this temporary now that we own the domain ?

File: _includes/header.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<header>
+  <img id="poutine" src="./images/poutine.png" title="CocoaHeads Montréal">
```
Comment (Palleas) [inline _includes/header.html:2]: Most Canadian code review ever

**Comment by Palleas** [comment]
Why do we keep the `current_website/` folder ?

File: _config.yml (hunk 2 above)
Comment (lucolivier-dumaisblais) [inline _config.yml]: Fixed to http://cocoaheadsmtl.com

**Pull request merged**